IAMOTZ/ievents
# #156977983 API Documentation

#### What does this PR do?
Documents the API with Swagger API-Documentation.
#### Description of Task to be completed?
Document API
#### How should this be manually tested?
- clone the app and change into the project directory
- start the app in dev mode(instructions in the readme)
- visit `http://localhost:3000/api-docs` in your browser
- the API documentation should appear there
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#156977983 
#### Questions:
None
